jersey-servers/src/main/java/com/palantir/remoting3/servers/jersey/TraceEnrichingFilter.java

```diff
@@ -49,7 +52,8 @@ public void filter(ContainerRequestContext requestContext) throws IOException {
                 .map(Resource::getPath)
                 .orElse("(unknown)");
 
-        String operation = requestContext.getMethod() + " " + path;
+        String operation = requestContext.getMethod() + " "
```

Comment:
I don't understand this change. `path` is now unused?

Reply:
Ah yes I missed https://github.com/palantir/http-remoting/pull/544, closing the PR